### PR TITLE
Dev cc kernel

### PR DESCRIPTION
**Pull request by akashkollipara**

### CODE REVIEW

**Comment by akashkollipara** [comment]
@pranjalchanda08 
run
```shell
$ git checkout development
$ git pull origin development
$ git checkout dev_cc_kernel
$ git rebase development
$ git push origin dev_cc_kernel -f
```

File: .vscode/settings.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
         "*.{cpp,cpp.dev,cpp.temp,cpp.tmp,cpp.old}": "C++",
         "*.{S,S.old,S.dev.S.temp,S.tmp,s.old,s.temp,s.tmp,s.dev,asm,asm.old,asm.dev.asm.temp,asm.tmp,inc,inc.old,inc.temp,inc.tmp,ld,ld.old,ld.temp,ld.tmp,ld.sx,ld.sx.old,ld.sx.temp,ld.sx.tmp,lst}": "coffeescript",
         "arch.h": "c",
-        "driver.h": "c"
+        "driver.h": "c",
+        "errno.h": "c"
```
Comment (akashkollipara) [inline .vscode/settings.json:44]: Please remove these changes... let them be in local repo

File: .github/workflows/github_ci.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@ name: GitHub CI
 
 on:
   push:
-    branches: [ stable, development ]
+    branches: [ stable, development, dev_cc_kernel ]
```
Comment (akashkollipara) [inline .github/workflows/github_ci.yml:5]: Remove this change... it wont be needed anymore.

File: projects/demo_avr/config.mk [hunk 3]
```diff
@@ -14,3 +14,4 @@ FAMILY		:= mega_avr
 PLATFORM	:= atmega328p
 EARLYCON_SERIAL	:= 1
 CONSOLE_SERIAL	:= 1
+TERRAKERN	:= 1
```
Comment (akashkollipara) [inline projects/demo_avr/config.mk:17]: make this 0

**Review by akashkollipara — COMMENTED** [review]
LGTM, @pranjalchanda08 please approve it.

**Review by pranjalchanda08 — APPROVED** [review]
LGTM!

**Pull request merged**